### PR TITLE
impl: Rename latest draft (v1.1) to draft

### PR DESCRIPTION
**Pull request by lehors**
Add clear disclaimer about the status of this document, and unhide it.

This closes issue #1086

### CODE REVIEW

File: docs/spec/draft/onepage.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 noindex: true
 ---
 {%- comment -%}
 A single page containing all the following files as different sections
 {%- endcomment -%}
 
-{% assign dir = "/spec/v1.1/" %}
-{% assign filenames = "whats-new,about,threats-overview,use-cases,principles,faq,future-directions,terminology,levels,requirements,distributing-provenance,verifying-systems,verifying-artifacts,threats,provenance,verification_summary" %}
+{% assign dir = "/spec/draft/" %}
+{% assign filenames = "whats-new,about,threats-overview,use-cases,principles,faq,future-directions,terminology,levels,requirements,distributing-provenance,verifying-systems,verifying-artifacts,threats,source-requirements,provenance,verification_summary" %}
```
Comment (zachariahcox) [inline docs/spec/draft/onepage.md:11]: is this generated somehow or manually kept in sync?

**Review by zachariahcox — APPROVED** [review]

**Review by TomHennen — APPROVED** [review]
Thanks!

File: docs/spec/draft/onepage.md (hunk 1 above)
Comment (lehors) [reply docs/spec/draft/onepage.md:11]: That part is manually managed.

Comment (lehors) [reply docs/spec/draft/onepage.md:11]: I created the liquid script to generate the single-view html page, fixing links to make them relative on the way, but didn't go any further. We could look into gathering the list of files from somewhere else but so far it hasn't changed often enough to motivate me to try and do that. :-)

**Review by mlieberman85 — APPROVED** [review]
Lgtm

**Pull request merged**